src/containers/SceaHomepageSpeciesContainer.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import withFetchLoader from './FetchLoader'
+import cardPropTypes from '../cards/cardPropTypes'
+import SpeciesCard from '../cards/SpeciesCard'
+
+const CardContainer = styled.div`
+  border-radius: 8px;
+  :hover {
+    background: AliceBlue;
+  }
+`
+
+const SceaHomepageSpeciesContainer = ({cards}) =>
```

Comment:
Similar comment as before, this could have a more generic name. It's basically a grid of species cards, regardless of where it is used in SC.